stable/kong/Chart.yaml

```diff
@@ -11,4 +11,4 @@ name: kong
 sources:
 - https://github.com/Kong/kong
 version: 0.5.2
-appVersion: 0.14.1
+appVersion: 0.14.1
```

Comment:
please add the new line as good practice

Reply:
new line has been added

Reply:
@shashiranjan84 which character I need to add in end of the line to pass this tests? Now I have added "\n" but tests were failed with `(error    too many blank lines (1 > 0)  (empty-lines))`